acceptance-tests/src/main/java/io/blueocean/ath/pages/classic/LoginPage.java

```diff
@@ -35,36 +35,24 @@ public LoginPage(WebDriver driver) {
     }
 
     public static String getUsername() {
-        return "alice";
+        return System.getProperty("adminUsername", "alice");
     }
 
     public static String getPassword() {
-        return "alice";
+        return System.getProperty("adminPassword", "alice");
     }
 
-    @Inject
-    WaitUtil wait;
     public void open() {
-        driver.get(base + "/login");
+        go(base + "/login");
         Assert.assertEquals(base + "/login", driver.getCurrentUrl());
     }
 
     public void login() {
         open();
-
-
-        WebElement usernameField = wait.until(By.id("j_username"));
-        usernameField.sendKeys(getUsername());
-
-        wait.until(By.name("j_password")).sendKeys(getPassword());
-
-        wait.until(By.xpath("//*/button[contains(text(), 'log')]")).click();
-        wait.until(driver -> {
-            if(driver.getCurrentUrl().contains("loginError")) {
-                throw new RuntimeException("Error logging in");
-            }
-            return ExpectedConditions.urlToBe(base + "/").apply(driver);
-        });
-        logger.info("Logged in as alice");
+        find("input[name=j_username]").sendKeys(getUsername());
```

Comment:
This was a 30 minute example of how my prior testing frameworks looked... well, similar.

Reply:
Only thing I would put here is that I would make find use "By" class. Ie find(By.id(..)), good otherwise. Could default to use css for strings though

Reply:
I suppose it doesn't hurt to have an override, but I would definitely not use `By` as the only way to specify things since you can specify a find by id using css like `#my-fancy-id`, which would work with this already. The only thing missing is find by link text, which you can use xpath for, but is a little more cumbersome. We really shouldn't be finding anything by text in these tests, anyway, tho